ovos_utils/dialog.py

```diff
@@ -129,8 +132,10 @@ def load_dialogs(dialog_dir, renderer=None):
     return renderer
 
 
-def get_dialog(phrase, lang=None, context=None):
-    """Looks up a resource file for the given phrase.
+def get_dialog(phrase: str, lang: str = None,
+               context: Optional[dict] = None) -> str:
+    """
+    Looks up a resource file for the given phrase.
```

Comment:
Maybe this should be updated to specify `lang` is required and fallback to configuration is deprecated (in a later release)

Reply:
#130